MailchimpSDK/MailchimpSDK.xcodeproj/project.pbxproj

```diff
@@ -560,6 +560,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MARKETING_VERSION = 0.1.5;
```

Comment:
New function adds `MARKETING_VERSION` to `Mailchimp-SDKTests`. I think I can use an argument to specify the `MailchimpSDK` Target only. Thoughts?

Reply:
IMO I think it is fine as-is.